package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sails-pager",
-  "version": "2.0.5",
+  "version": "2.0.6",
```

Comment:
@segsalerty2013 Nice one here. The version should now be 2.1.0 since this is a new feature, i'll merge immediately you update.